tests/benchmarks/bonnie_benchmark_test.py

```diff
@@ -30,7 +33,8 @@ def setUp(self):
       self.contents = fp.read()
 
   def testParseCSVResults(self):
-    result = bonnie_benchmark.ParseCSVResults(self.contents)
+    with mock.patch(time.__name__ + '.time', return_value=1.0):
```

Comment:
Maybe consider:
* adding a [custom type equality function](https://docs.python.org/2/library/unittest.html#unittest.TestCase.addTypeEqualityFunc) which ignores timestamp
* patching time.time in setUp
* a custom assert method.